src/AWS/Signers/V4.elm

```diff
@@ -42,8 +42,7 @@ algorithm =
 
 headers : AWS.Config.Service -> List ( String, String )
 headers config =
-    [ ( "Host", AWS.Http.host config.endpoint )
-    , ( "Accept", "application/json" )
+    [ ( "Accept", "application/json" )
```

Comment:
removing host from the list of headers that we try to set. Fixes the error we were seeing in the browser log, but breaks signing so...